src/advanced_inv.cpp

```diff
@@ -1901,7 +1901,7 @@ bool advanced_inv_area::is_same( const advanced_inv_area &other ) const
         id != AIM_WORN      && other.id != AIM_WORN      &&
         id != AIM_CONTAINER && other.id != AIM_CONTAINER ) {
 
-        if( veh == other.veh ) {
+        if( veh != nullptr && veh == other.veh ) {
```

Comment:
I'm playing with this right now and this doesn't seem to be the case. I can select ALL in a pane, then in the other select a vehicle part and the items in the vehicle disappear from the ALL pane. Was there some other situation that would cause this I can test?